### ISSUE TITLE
Scrolling empty list causes crash

### ISSUE THREAD

**Issue by Lavmint** [JoniVR/VerticalCardSwiper#28]
### New Issue Checklist

- [x] I updated VerticalCardSwiper to the latest version.
- [x] I read the [Contribution Guidelines](CONTRIBUTING.md).
- [x] I read the [documentation](https://github.com/JoniVR/VerticalCardSwiper).
- [x] I searched for [existing GitHub issues](https://github.com/JoniVR/VerticalCardSwiper/issues).

### Issue Description

Steps to reproduce:
1. return numberOfCards equal to 0
2. try to scroll
3. Fatal error: Unexpectedly found nil while unwrapping an Optional value of `swipeAbleArea` in `VerticalCardSwiper.swift` on line `258`

Temporary solution for other developers:
- disable user interaction if numberOfCards equal to 0
### Environment

- **iOS Version**: iOS 12.1
- **Device(s)**: iPhone 5S Simulator
- **Xcode Version**: 10.1 (10B61)

**Comment by JoniVR** [comment]
Hi @Lavmint!

Thanks for reporting this bug.
This has been fixed a while ago (in commit ac70a99 of development branch) but isn't released yet, which is why it's still occurring at the moment for you.

I'm currently very busy with other projects and school work so I can't really spend too much time on this project at the moment.

If you really need the fix right now, you can use the following line in your Podfile to use the `development` branch for now:
```rb
pod 'VerticalCardSwiper', :git => 'https://github.com/JoniVR/VerticalCardSwiper.git', :branch => 'development'
```

(keep in mind that some other new stuff in there is still experimental)

Hope this helps, I'll be closing this as it's already been fixed, but feel free to reopen if you have any further remarks or questions.

Best regards and happy holidays!
JoniVR

**edit:** 
Alternatively, you could also point to the specific commit in your podfile for now, which might be a better option:
```rb 
pod 'VerticalCardSwiper', :git => 'https://github.com/JoniVR/VerticalCardSwiper.git', :branch => 'development', :commit => 'ac70a99cb2'
```
(I haven't tested this but it should in theory work).

**Comment by JoniVR** [comment]
Hi, just letting you know that 0.1.0-beta7 will be released in a few minutes, which fixes the issue, so you can just point to that instead of development branch 😊
